data/human/human missions.txt

```diff
@@ -3833,3 +3832,158 @@ mission "Spaceport Reminder"
 			`	He pleasantly smiles and nods. "Well, I wish you the best in your travels." He walks away, leaving you free to explore the rest of <origin>'s facilities.`
 				decline
 
+
+
+mission "Fumbling Mugger Patch"
+	landing
+	invisible
+	to offer
+		has "Transport Workers B: done"
+	on offer
+		event "fumbling mugger timer" 60
+		fail
+
+
+event "fumbling mugger timer"
+
+
+mission "Fumbling Mugger"
+	name "Transport to <planet>"
+	description "Bring a young woman who threatened you on <origin> to <destination>."
+	minor
+	source
+		attributes "dirt belt" "urban"
+	destination "Maker"
+	passengers 1
+	to offer
+		random < 45
+		has "event: fumbling mugger timer"
+	on offer
+		conversation
+			`Your exploration of the spaceport leads you to a dark alley where the number of working lights are far outnumbered by the number of broken ones. The contents of steel bins lining the walls overflow onto the pavement, attracting swarms of flies to them. On the surfaces that are illuminated, you can see a mess of black scrawlings upon bare, cracked concrete, and the few windows present on the towering walls surrounding you are either boarded up or shattered. A dark crimson stains the bottom half of a broken window.`
+			`	Suddenly, you feel a cold object pushed against your back as a woman speaks softly. "Hands where I can see them, Captain."`
+			choice
+				`	(Lift your hands up and slowly turn.)`
+					goto peaceful
+				`	(Spin around and fire.)`
+				`	(Try a fancy acrobatic move and disarm your attacker.)`
+					goto flip
+			
+			`	You turn around as swiftly as you can as your hand grasps for your pistol. Due to surprise, or perhaps nervousness, the woman threatening you fails to fire the shotgun she's holding in the second you spend turning. You manage to land a shot, and your quarry falls while a cry escapes from her body. As you walk away, her body fades into the darkness, blending with the garbage bags around it.`
+				decline
+			
+			label flip
+			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, you feel metal dig into your spine as a gasp escapes from behind you, before landing on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`
```

Comment:
```suggestion
			`	You push down as hard as you can with your legs while leaning backwards, rising to the lofty height of two feet above the ground. As you begin to fall back-first uncontrollably, cold metal digs into your spine. A gasp escapes from behind as you land on a soft body. You pick yourself up as fast as possible, and you end up face-to-face with your attacker. In the darkness, you pick out that she's wearing a black hoodie with a bandanna over her mouth, her back hunched over and feet placed apart. As you look down, you spot a sawed-off break-action shotgun held single-handed, the end of which is pointed at your abdomen. "Hands up now, Captain."`
```
That `two feet` bit made me chuckle :)
I feel like that middle sentence could use some work - feels very long and messy as-is. If you've got any alternative ideas than the ones I'm suggesting, be my guest!